dll/win32/shlwapi/utils.cpp

```diff
@@ -159,19 +177,36 @@ IShellFolder_GetDisplayNameOf(
     if (!SHLWAPI_IsBogusHRESULT(hr))
         return hr;
 
-    dwRetryFlags |= 0x80000000;
+    dwRetryFlags |= SFGDNO_RETRYALWAYS;
 
     if ((uFlags & SHGDN_FORPARSING) == 0)
         dwRetryFlags |= SFGDNO_RETRYWITHFORPARSING;
 
-    /* It seems the function is actually retrying here */
-    FIXME("dwRetryFlags: 0x%X\n", dwRetryFlags);
+    // Retry with another flags
```

Comment:
```suggestion
    // Retry with other flags
```
Can you tell us the purpose of this retry?

Reply:
Fixed. Added the purpose description.